.github/workflows/gcp-test-deploy.yml

```diff
@@ -0,0 +1,315 @@
+name: Test instance deployment
+
+on:
+  workflow_call:
+    inputs:
+      network:
+        required: false
+        type: string
+      app_name:
+        required: true
+        type: string
+      test_id:
+        required: true
+        type: string
+      test_name:
+        required: true
+        type: string
+      test_description:
+        required: true
+        type: string
+      test_variable:
+        required: true
+        type: string
+      zebra_state_path:
+        required: true
+        type: string
+      disk_prefix:
+        required: false
+        type: string
+      disk_suffix:
+        required: false
+        type: string
+      needs_constants:
+        required: true
+        type: boolean
+      builds_disks:
+        required: true
+        type: boolean
+      height_grep_text:
+        required: false
+        type: string
+
+env:
+  NETWORK: Mainnet
+  IMAGE_NAME: zebrad-test
+  GAR_BASE: us-docker.pkg.dev/zealous-zebra/zebra
+  ZONE: us-central1-a
+  MACHINE_TYPE: c2d-standard-16
+
+jobs:
+  build:
+    uses: ./.github/workflows/docker-image-build.yml
```

Comment:
Bug: `reusable workflow cannot be nested`

Reply:
Fixed by: [fix(rw): remove nested worflow](https://github.com/ZcashFoundation/zebra/pull/4271/commits/3b6c87cf0cbdd99e2fccf9d07721208471f7e790)